composer.json

```diff
@@ -7,7 +7,8 @@
     },
     "require": {
         "filp/whoops": "^1.1 || ^2.0",
-        "zendframework/zend-diactoros": "^1.1.0"
+        "zendframework/zend-diactoros": "^1.1.0",
+        "http-interop/http-middleware": "^0.4.1"
```

Comment:
Please don't add this here, as it is optional. Instead, add it to `suggest` and `require-dev` (as we should also add tests?).